packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -121,7 +121,8 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
     onGoToStep: null as WizardStepFunctionType,
     width: null as string,
     height: null as string,
-    footer: null as React.ReactNode
+    footer: null as React.ReactNode,
+    onClose: () => undefined as any
```

Comment:
Can we do this instead for consistency?
```suggestion
    onClose: (): any  => undefined 
```

Reply:
Right now there's 11 results for `): any => undefined` and 22 for `) => undefined as any`. Are we preferring the former?

Reply:
We should fix these consistency issues and decide on which format we would like to use.  Could add some tslint rules for things when possible.  I would recommend leaving this for now and fixing some of these consistency issues in a separate issue.